### DISCUSSION TITLE
简悦导入到 flomo 的几种常见方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2150] [服务]
> 简悦 2.2 开始接入了 flomo，可以让简悦更方便的归纳你的 flomo 以下是基于简悦导入 flomo 的几种常见方案。

## 基础用法

如果某个页面有多个需要摘录的内容，你可以使用简悦导入到 flomo 的方案，特点优势：支持简悦的 标题 / 描述 / 备注 / 标签 / 标注，如下图，是在简悦的稍后读效果：

![image](https://user-images.githubusercontent.com/81074/119644346-57395100-be4f-11eb-803a-10dbaea70fcf.png)

当你导入到 flomo 后会发现如下效果：

![image](https://user-images.githubusercontent.com/81074/119644490-7cc65a80-be4f-11eb-8c22-216fa30cab5f.png)

你会惊奇的发现，在简悦中的标签完美的出现在了 flomo 中。 这完全是一个「不谋而合」，冥冥之中简悦跟 flomo 的关系从这里开始的...

> 因为 flomo 的标签系统与简悦完全一致。

### 为什么无法导入全文到 flomo ？

在简悦接入 flomo 之前，我就跟少楠老师讨论过这个事情，只因 flomo 是一个灵感类记录 App，所以本质上很排斥对全文的摘录。再者说，因为 flomo 的轻巧特性，也无法支持 Web 正文中的各种格式，所以当没有标注时，仅就摘录了页面最基本信息，如下图：

![image](https://user-images.githubusercontent.com/81074/119646183-72a55b80-be51-11eb-8d7f-e6c71f26fb0d.png)

> 上面的只是基础玩法，接下来就该 Level Up 了。

## 进阶用法

**[简悦 2.2.0](http://ksria.com/simpread/welcome/version_2.2.0.html)** 增加了对信息的整理，如：双向链接，知识图谱，支持：Evergreen Notes / Workflowy / Kanban 布局，[详见这里](http://ksria.com/simpread/welcome/version_2.2.0.html#unreader) 。

利用双向链接特性，可以在将不同的页面的标注（摘录）内容归于一处，再统一导入到 flomo 做二次整理。

比如前阵子我对 **古代的北方少数民族** 这个内容比较感兴趣，所以在维基百科上面找到了很多相关性的内容，通过反向链接，将它们链接到一起，如下图：

![image](https://user-images.githubusercontent.com/81074/119646776-1e4eab80-be52-11eb-98fa-4a6897c4ccd1.png)

> 备注：上图的内容需要自己使用反向链接的方式慢慢积累，此过程略去。

此页面也支持导入到 flomo，导入后的效果如下图：

![Xnip2021-05-26_18-45-04](https://user-images.githubusercontent.com/81074/119647439-d9774480-be52-11eb-9acf-7557c7377a8a.png)

备注1：上图的内容稍作整理（去掉了一些多余回车）

备注2：在简悦的稍后读整理的越精细，导出的内容也就越丰富。

### 这么做的优势？

1. 简悦更适合在 Web 页面标注内容。
2. 因为简悦 2.2.0 的特性：双向链接，它比标签具有更灵活的特性，可以将不同的信息归结于一处。
3. 因为简悦 2.2.0 的特性：知识图谱，所以在稍后读中可以通过更直观的方式查看。（如下图

![](https://z3.ax1x.com/2021/05/16/ggmTht.gif)

## 高级用法

上面的例子只是能算经典案例，现在来看看进阶的案例。

你可以通过 [Workflowy 布局](http://ksria.com/simpread/docs/#/%E7%A8%8D%E5%90%8E%E8%AF%BB-%E5%A4%9A%E7%A7%8D%E5%B8%83%E5%B1%80?id=workflowy)，将你认为需要导出的内容一键导入到 flomo。

一个例子：**古代的北方少数民族**，你收集了很多内容，如下图：

![2021-05-26_19-01-37 (1)](https://user-images.githubusercontent.com/81074/119649266-0593c500-be55-11eb-8bb8-ec191e9a6e21.gif)

> 导入到 flomo 后，效果如下图：

![Xnip2021-05-26_19-05-03](https://user-images.githubusercontent.com/81074/119649595-63281180-be55-11eb-96ed-932f53bbdd7a.png)

> 备注：截图的结构稍作调整，删除了一些无用回车。

## 疯狂玩家

1. 看着是不是感觉不错？这还不算什么，如果你愿意，可以将 [当简悦变成你的知识库（离线且具有永久链接）](https://github.com/Kenshin/simpread/discussions/2221) 让上面的这些信息变为永久信息。

2. [定制导入到 flomo 的模板](https://github.com/Kenshin/simpread/discussions/2275)

**Comment by hezhefly** [comment]
啥呀，触不及防的放了个鸽子

**Reply by Kenshin** [reply]
我也不想… 😂
更新后会通知你。
